pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -399,9 +402,118 @@ func (r *rbac) createRoleBinding(ctx context.Context, sa *corev1.ServiceAccount)
 		Subjects: []rbacv1.Subject{{Kind: rbacv1.ServiceAccountKind, Name: sa.Name, Namespace: sa.Namespace}},
 	}
 
-	_, err = rbacClient.RoleBindings(sa.Namespace).Create(ctx, rb, metav1.CreateOptions{})
-	if err != nil {
+	if _, err := rbacClient.RoleBindings(sa.Namespace).Create(ctx, rb, metav1.CreateOptions{}); err != nil {
 		logger.Error(err, "creation of 'edit' rolebinding failed, in Namespace", sa.GetNamespace())
+		return err
+	}
+	return nil
+}
+
+func (r *rbac) ensureClusterRoleBindings(ctx context.Context, sa *corev1.ServiceAccount) error {
+	logger := logging.FromContext(ctx)
+
+	rbacClient := r.kubeClientSet.RbacV1()
+	logger.Info("finding cluster-role clusterinterceptors-clusterrole")
+	if _, err := rbacClient.ClusterRoles().Get(ctx, "clusterinterceptors-clusterrole", metav1.GetOptions{}); errors.IsNotFound(err) {
+		return r.createClusterRole(ctx)
+	}
+
+	logger.Info("finding cluster-role-binding clusterinterceptors-clusterrolebinding")
+
+	viewCRB, err := rbacClient.ClusterRoleBindings().Get(ctx, "clusterinterceptors-clusterrolebinding", metav1.GetOptions{})
```

Comment:
let us add openshift-pipelines- prefix to this name and
we can skip the clusterrole suffix, as it is redundant.